src/Menus/Settings.js

```diff
@@ -182,5 +182,4 @@ const Settings = () => {
         </Card>
     )
 }
-
```

Comment:
Check For the if return an undefined value:-  After the token is invalid this function automatically becomes undefined and causes the error  

u can use this or any alternative
return value.menu_slno ?? 0  ;

getMenuSlno().then((val) => {
            const menuSlnoArray = val.map((value) => {
                return value.menu_slno;
            })